mlir/test/fusion/pr-e2e/reductions/atomic_add/lit.local.cfg

```diff
@@ -1,2 +1,2 @@
-if not 'atomic_add' in config.features:
+if not config.enable_rock_driver_pr_e2e_test or config.no_AMD_GPU or not 'atomic_add' in config.features:
```

Comment:
If there is no gpu, shouldn;t we have an empty config.features ?

Reply:
The problem is is that if there's no GPU, then `config.features` is `None`, which means `'atomic_add' in config.features` is `'atomic_add` in None` which throws an exception and prevents any tests from running at all

Maybe the fix is changing how `config.features` is defined, but a `no_AMD_GPU` check is more in line with the rest of the test suite

Reply:
I see ok. I thought no_AMD_GPU is thing introduced in this PR; it was there...